pkg/monitoring/source_envelope.go

```diff
@@ -162,23 +173,32 @@ func (s *envelopeSource) getJuelsPerLamport(ctx context.Context) (*big.Int, erro
 			},
 		)
 		if err != nil {
-			return nil, fmt.Errorf("failed to fetch tx with signature %s: %w", txSig.Signature, err)
+			s.log.Infow("failed to fetch tx with signature %s: %w", txSig.Signature, err)
```

Comment:
Not fully sure, but shouldn't this be `s.log.Infof` instead of `s.log.Infow`? See https://pkg.go.dev/go.uber.org/zap#SugaredLogger.Infow

Reply:
Indeed `Infow` is supported by the logger but according to [the implementation](https://pkg.go.dev/go.uber.org/zap#SugaredLogger.Infow) `Infow` is meant to use with keyValues as args, while `Infof` is meant to be used with `fmt`-formatted messages. The latter is being used here with `%s` and `%w`, so that's why I expected `Infof`.
However, if it works fine then there's no reason to spend effort on changing this.

Reply:
Ah yes. You're right! It works as is but it's not ideal, the values are not keyed correctly. I'll fix

Reply:
Done. Btw, do you know what this failure is https://github.com/smartcontractkit/chainlink-solana/actions/runs/3872239678/jobs/6602413818 I'm not sure I'm causing it!

Reply:
No clue! When I run `./scripts/install-solana-ci.sh` locally it works properly:

```
<snip>
} # this ensures the entire script is downloaded #'
downloading v1.10.33 installer
  ✨ 1.10.33 initialized
<snip>
```

Reply:
it passes now 🤷